### PR TITLE
Atlassian template for installing the Sumologic atlassian solution.

### PR DESCRIPTION
**Pull request by arunpatyal**

### CODE REVIEW

**Comment by ankitgoelcmu** [comment]
LGTM

File: atlassian-terraform/terraform.tfvars [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Sumo Logic - Atlassian Terraform
```
Comment (duchatran) [inline atlassian-terraform/terraform.tfvars:1]: I am a bit worried about this file - there are so many variables here, easy to miss one. For all MANDATORY variables or things people need , can we use a syntax like <> and mention that people need to provide values for those places, INCLUDING the angle brackets

For example: 
environment ="<deployment>"  #  please provide the deployment for your Sumo org

File: atlassian-terraform/terraform.tfvars [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+# Configure Sumo Logic, Jira Cloud, Jira Server (Onprem), OpsGenie, BitBucket credentials and installation options.
+
+# Sumologic
+sumo_access_id = "" # https://help.sumologic.com/Manage/Security/Access-Keys
```
Comment (duchatran) [inline atlassian-terraform/terraform.tfvars:5]: To continue my above message: 
`sumo_access_id =  "<YOUR SUMO ACCESS ID>" # Please replace <YOUR SUMO ACCESS ID> (including brackets) with your Sumo Access ID`

File: atlassian-terraform/terraform.tfvars [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+install_opsgenie = "true"
+# install_opsgenie should be true for the below option install_sumo_to_opsgenie_webhook to be true. https://help.sumologic.com/Manage/Connections-and-Integrations/Webhook-Connections/Webhook_Connection_for_Opsgenie
+install_sumo_to_opsgenie_webhook = "false" # This feature is in Beta. To participate contact your Sumo account executive. You can modify the file sumo_to_opsgenie_webhook.json.tmpl for customizing payload.
+install_atlassian_app = "true"
+install_sumo_to_jiracloud_webhook = "false" # This feature is in Beta. To participate contact your Sumo account executive. You can modify the file sumo_to_jiracloud_webhook.json.tmpl for customizing payload.
+install_sumo_to_jiraserver_webhook = "false" # This feature is in Beta. To participate contact your Sumo account executive. You can modify the file sumo_to_jiraserver_webhook.json.tmpl for customizing payload.
+
+# Jira Cloud
+jira_cloud_url = "https://<example>.atlassian.net"
+jira_cloud_user = ""
```
Comment (duchatran) [inline atlassian-terraform/terraform.tfvars:24]: Is this mandatory? If so use something like: 
`jiar_cloud_user = "<cloud username>" # Provide your JIRA username`

File: atlassian-terraform/terraform.tfvars [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+"sprint_created","sprint_deleted","sprint_updated","sprint_started","sprint_closed",
+"board_created","board_updated","board_deleted","board_configuration_changed",
+"jira_expression_evaluation_failed"]
+
+# Sumologic to Jira Cloud Webhook - This feature is in Beta. To participate contact your Sumo account executive.
+# https://developer.atlassian.com/cloud/jira/platform/rest/v2/#api-rest-api-2-issue-post
+jira_cloud_issuetype = "Bug"
+jira_cloud_priority = "3"
+jira_cloud_projectkey = "project"
+jira_cloud_auth = "" #Basic Authorization Header. See: https://help.sumologic.com/Beta/Webhook_Connection_for_Jira_Cloud#prerequisite
```
Comment (duchatran) [inline atlassian-terraform/terraform.tfvars:45]: Similarly, is this a required parameter, or an empty string is the default value?

File: atlassian-terraform/terraform.tfvars [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+# Jira On Prem/Server
+# This script configures Jira Server WebHooks. Jira Server Logs collection needs to be configured as explained in Step 1 [here](https://help.sumologic.com/07Sumo-Logic-Apps/08App_Development/Jira/Collect_Logs_for_Jira#step-1-set-up-local-file-sources-on-an-installed-collector).
+# Configure the log collection and update the variable jira_on_prem_access_log_category with the selected source category.
+jira_on_prem_access_logs_sourcecategory = "Atlassian/Jira/Server*"
+jira_on_prem_url = ""
```
Comment (duchatran) [inline atlassian-terraform/terraform.tfvars:51]: `jira_on_prem_url = "<your JIRA onprem URL> "`

File: atlassian-terraform/terraform.tfvars [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+jira_on_prem_user = ""
```
Comment (duchatran) [inline atlassian-terraform/terraform.tfvars:52]: `jira_on_prem_user = "<your JIRA onprem User>"`

File: atlassian-terraform/terraform.tfvars [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+jira_on_prem_password = "" # Needs to be the password. API Key is not supported on Jira Server yet: https://jira.atlassian.com/browse/JRASERVER-67869?_ga=2.198461357.302520551.1583314185-1454539139.1580206139&error=login_required&error_description=Login+required&state=d3142ec3-6eb1-4207-bdd7-ce6b93900aa1
```
Comment (duchatran) [inline atlassian-terraform/terraform.tfvars:53]: Same here

File: atlassian-terraform/terraform.tfvars [hunk 8]
```diff
@@ -0,0 +1,93 @@
+# Sumo Logic - Atlassian Terraform
+# Configure Sumo Logic, Jira Cloud, Jira Server (Onprem), OpsGenie, BitBucket credentials and installation options.
+
+# Sumologic
+sumo_access_id = "" # https://help.sumologic.com/Manage/Security/Access-Keys
+sumo_access_key = ""
+environment = "us1" # https://help.sumologic.com/APIs/General-API-Information/Sumo-Logic-Endpoints-and-Firewall-Security
```
[the remark this answers is not in the document]
Comment (arunpatyal) [reply atlassian-terraform/terraform.tfvars:7]: Changed to deployment. Sumologic terraform provider uses environment terminology so had kept the same but as you mentioned deployment makes more sense.

[the remark this answers is not in the document]
Comment (arunpatyal) [reply atlassian-terraform/terraform.tfvars:7]: Updated description as well.

File: atlassian-terraform/terraform.tfvars (hunk 1 above)
Comment (arunpatyal) [reply atlassian-terraform/terraform.tfvars:1]: Added.

File: atlassian-terraform/terraform.tfvars (hunk 2 above)
Comment (arunpatyal) [reply atlassian-terraform/terraform.tfvars:5]: Added

File: atlassian-terraform/terraform.tfvars (hunk 4 above)
Comment (arunpatyal) [reply atlassian-terraform/terraform.tfvars:45]: Added

File: atlassian-terraform/terraform.tfvars (hunk 3 above)
Comment (arunpatyal) [reply atlassian-terraform/terraform.tfvars:24]: Added

File: atlassian-terraform/terraform.tfvars (hunk 5 above)
Comment (arunpatyal) [reply atlassian-terraform/terraform.tfvars:51]: Added

File: atlassian-terraform/terraform.tfvars (hunk 6 above)
Comment (arunpatyal) [reply atlassian-terraform/terraform.tfvars:52]: Added

File: atlassian-terraform/terraform.tfvars (hunk 7 above)
Comment (arunpatyal) [reply atlassian-terraform/terraform.tfvars:53]: Added

**Pull request merged**